modules/security-best-practices/pages/rust-canister-development-security-best-practices.adoc

```diff
@@ -34,7 +34,7 @@ If this is not the case, an attacker may be able to perform sensitive actions on
 
 ==== Recommendation
 
-- By design, for every canister call the caller can be identified. The calling link:https://smartcontracts.org/docs/interface-spec/index.html#principal[principal] can be accessed using the system API’s methods `ic0.msg_caller_size` and `ic0.msg_caller_copy` (see link:https://docs.dfinity.systems/spec/public/#system-api-imports[here]). If e.g. Internet Identity is used, the principal is the user identity for this specific origin, see link:https://github.com/dfinity/internet-identity/blob/main/docs/internet-identity-spec.adoc#identity-design-and-data-model[here]. If some actions (e.g. access to user’s account data or account specific operations) should be restricted to a principal or a set of principals, then this must be explicitly checked in the canister call, for example as follows in Rust:
```

Comment:
This change might have been unintended. Rest all looks good to me.

Reply:
Oh, thanks for pointing it out! My search/replace was too naive!

Fixed it.